x-pack/plugins/triggers_actions_ui/server/plugin.ts

```diff
@@ -53,14 +53,15 @@ export class TriggersActionsPlugin implements Plugin<void, PluginStartContract>
       plugins.alerting !== undefined
     );
 
-    core.getStartServices().then(([_, pluginStart]) => {
-      createConfigRoute({
-        logger: this.logger,
-        router,
-        baseRoute: BASE_TRIGGERS_ACTIONS_UI_API_PATH,
-        alertingConfig: plugins.alerting.getConfig,
-        getRulesClientWithRequest: pluginStart.alerting.getRulesClientWithRequest,
-      });
+    createConfigRoute({
+      logger: this.logger,
+      router,
+      baseRoute: BASE_TRIGGERS_ACTIONS_UI_API_PATH,
+      alertingConfig: plugins.alerting.getConfig,
+      getRulesClientWithRequest: async (request) => {
+        const [, pluginStart] = await core.getStartServices();
+        return await pluginStart.alerting.getRulesClientWithRequest(request);
```

Comment:
This `await` is not needed - the `pluginStart.alerting.getRulesClientWithRequest(()` is not async.  In fact, I have a little VSCode squiggle under it:

>  'await' has no effect on the type of this expression.ts(80007)

Reply:
Good catch! Fixed in [`907e7a2` (#181456)](https://github.com/elastic/kibana/pull/181456/commits/907e7a2adec199094b0f562e9902f6bfa33daefd).